.babelrc

```diff
@@ -9,6 +9,7 @@
   "plugins": [
     "@babel/proposal-class-properties",
     "@babel/proposal-object-rest-spread",
-    "@babel/plugin-transform-runtime"
+    "@babel/plugin-transform-runtime",
+    "@babel/plugin-proposal-export-default-from"
```

Comment:
good question, I don't see that we're actually using the export default from syntax.
I added it after getting errors building the eth2.0-xx packages, which said to install the plugin. I don't remember what the exact error was though.

I'll see if it can be removed, or what the error is.

Reply:
removed